### DISCUSSION TITLE
No more -binaries.7z release?

### DISCUSSION THREAD

**Discussion by rom1v** [community/community#110565] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

For libusb 1.0.26, you provided [`libusb-1.0.26-binaries.7z`](https://github.com/libusb/libusb/releases/download/v1.0.26/libusb-1.0.26-binaries.7z). It was very convenient, there was already the pkgconfig file, with the correct directory hierarchy ready to add libusb as a dependency in meson for cross-compiling.

I use it in my release [script](https://github.com/Genymobile/scrcpy/blob/ffa238b9d35bb9c882537d32724bbadbe4da7ef6/app/prebuilt-deps/prepare-libusb.sh) so that libusb is available in [meson](https://github.com/Genymobile/scrcpy/blob/ffa238b9d35bb9c882537d32724bbadbe4da7ef6/app/meson.build#L114).

There is no such release in [libusb 1.0.27](https://github.com/libusb/libusb/releases/tag/v1.0.27). Any particular reason?

---

EDIT: Oops, sorry, I followed a link from the libusb repo, I thought it pointed to the discussion section of the repo. Sorry for that.

**Comment by ettaboyle** [comment]
Hi @rom1v,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's repository](https://github.com/zigbee2mqtt). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

Best of luck!
